client/src/components/TrackGroup/TrackGroup.tsx

```diff
@@ -286,20 +301,43 @@ function TrackGroup() {
         </div>
         <div
           className={css`
-            margin: 1.25rem 0;
-            padding: 0.5rem 0 0.25rem;
+            display: flex;
+            @media screen and (max-width: ${bp.small}px) {
+              flex-direction: column;
+            }
           `}
         >
-          <MarkdownContent content={trackGroup.about} />
-        </div>
+          <div
+            className={css`
+              max-width: 70%;
+              margin: 1.25rem 0 1.25rem;
+              padding: 0.5rem 2rem 0.25rem 0rem;
+              border-right: 1px solid;
+              @media screen and (max-width: ${bp.small}px) {
+                max-width: 100%;
+                margin: 1.25rem 0 1.25rem;
+                padding: 0.5rem 0rem 0.25rem 0rem;
+                border-right: 0;
+              }
+            `}
+          >
+            <MarkdownContent content={trackGroup.about} />
+          </div>
 
-        <div
-          className={css`
-            margin: 1.25rem 0;
-            padding: 0.5rem 0.25rem;
-          `}
-        >
-          <MarkdownContent content={trackGroup.credits} />
+          <div
+            className={css`
+              margin: 1.25rem 0;
+              padding: 0.5rem 0.25rem 0.5rem 2rem;
+              font-size: var(--mi-font-size-small);
+              color: var(--mi-lighter-foreground-color);
```

Comment:
I feel like the `lighter` foreground colors should also be changed into transparency placed values